styles/_dropdowns.scss

```diff
@@ -47,13 +47,12 @@
   }
 
   [role="separator"] {
-    border-bottom: 1px solid rgb(216, 216, 216);
```

Comment:
Nit: why do we change this colour? Looks like we might it slightlyyyy lighter, right? Is that intentional?

Reply:
Good question. The color should match the design in Figma https://www.figma.com/file/6g87L4FdKZTA3knt3Rsfdx/Garden?node-id=254%3A1241 
which is #e9ebed. If anything we should change it to that color.